src/test/regress/multi_schedule

```diff
@@ -173,7 +173,8 @@ test: multi_modifications
 test: multi_distribution_metadata
 test: multi_generate_ddl_commands multi_create_shards multi_prune_shard_list multi_repair_shards 
 test: multi_upsert multi_simple_queries multi_create_insert_proxy multi_data_types
-test: multi_utilities foreign_key_to_reference_table validate_constraint
```

Comment:
Setting different values to `citus.next_shard_id` and `citus.next_placement_id` in `foreign_key_to_reference_table` and `validate_constraint` tests are not causing any problem in my local, but I am guessing they are the main reason why one of these two tests fails occasionally on CI.

Either CI gives a timeout in this parallel group, or we have a different number of rows printed in a test query